core/example/benchmark/CMakeLists.txt

```diff
@@ -2,9 +2,3 @@ if(Cabana_ENABLE_Serial)
   add_executable(NeighborListMDPerfTest md_neighbor_perf_test.cpp)
   target_link_libraries(NeighborListMDPerfTest cabanacore)
 endif()
-
-# PeakFlops uses x86 asm volatile
-if(CMAKE_SYSTEM_PROCESSOR MATCHES "([xX]86|[aA][mM][dD]64)")
```

Comment:
Does it work on Power9 now?

Reply:
Nope. I moved that check down into `core/performance_test/peakflops/CMakeLists.txt `, but maybe it should be higher? 

My thinking was "only peakflops needs this check, so it should be as close to that as possible" (i.e some performance tests might work on p9)